src/main/java/homework_1/Main.java

```diff
@@ -1,9 +1,23 @@
 package homework_1;
 
 public class Main {
-
   public static void main(String[] args) {
-    System.out.println("Hello homework!");
+    int k = args.length;
+    if (k==0) {
+      System.out.println("\033[32mApplication has been called \033[31mwithout\033[32m the arguments!!! \033[0m");
```

Comment:
Better move escape strings to constants and use concatenation

Reply:
done